### PR TITLE
Minimum element from stack in O(1)

### PR DESCRIPTION
**Pull request by Aarti002**
issue #916 solved
@tarun26091999  Please review and merge my code 
Thank you!

### CODE REVIEW

File: Stack/Python/minimum_ele.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+#                 O(1) ---->poping an element ,O(n) for deleting any particuler element
+#                 O(1)  --->getting minimum element from stack
+
+#Space Complexity : O(n)
+
+#e.g
+#Wanna create a stack?(y/n)y
+#Menu:
+#0.Quit process
+#1.Push into stack
```
Comment (krishna-NIT) [inline Stack/Python/minimum_ele.py:123]: Do Proper Editing here, Give Proper Space and Tabs Here, Make it Readable

File: Stack/Python/minimum_ele.py [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+#Enter data: 11
+#Menu:
+#0.Quit process
+#1.Push into stack
+#2.Delete from stack
+#3.Print data
+#4.Get minimum
+#1
+#Enter data: 12
+#Menu:
```
Comment (krishna-NIT) [inline Stack/Python/minimum_ele.py:137]: Do Proper Editing here, Give Proper Space and Tabs Here, Make it Readable
Do this Everywhere in this Comments

**Comment by krishna-NIT** [comment]
@Aarti002  Update the Readme which is present at the location (Algo-Tree/Stack/readme.md
) , add this question this at Last of this Readme and arrange these question in alphabetical Order

**Comment by Aarti002** [comment]
@krishna-NIT can you review again , i guess i have made required changes.
Thank you!

**Comment by krishna-NIT** [comment]
Resolve the conflict

File: Stack/readme.md [hunk 3]
```diff
@@ -23,4 +23,4 @@ element in a full stack is called **overflow**.
 
 ## Questions :
 * Minimum Element ----> [Python](/Code/Python/minimium_element.py)
-* Stack Class ----> [C++](/Code/C++/stack_class.cpp)
+
```
Comment (krishna-NIT) [inline Stack/readme.md:26]: Don't Remove Previous Questions

File: Stack/Python/minimum_ele.py (hunk 2 above)
Comment (krishna-NIT) [reply Stack/Python/minimum_ele.py]: DO this changes @Aarti002

File: Stack/Python/minimum_ele.py [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+                else:
+                    print("Oops! stack is empty")
+    print('Okay bye there!')
+
+
+#Time comlpexity: O(1) ---->inserting an element
+#                 O(1) ---->poping an element ,O(n) for deleting any particuler element
+#                 O(1)  --->getting minimum element from stack
+
+#Space Complexity : O(n)
```
Comment (krishna-NIT) [inline Stack/Python/minimum_ele.py]: add what is n there?

**Comment by krishna-NIT** [comment]
@rish-singhal LGTM

**Comment by rudrakshi99** [comment]
Please resolve merge conflicts.

File: Code/Python/minimum_element.py [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+Wanna create a stack?(y/n)y
+Menu:
+0.Quit process
+1.Push into stack
+2.Delete from stack
+3.Print data
+4.Get minimum
+1
+Enter data: 11
+Menu:
```
Comment (krishna-NIT) [inline Code/Python/minimum_element.py:130]: Format Test Cases Properly

**Review by krishna-NIT — APPROVED** [review]

**Review by rudrakshi99 — APPROVED** [review]

**Pull request merged**